Change "control_socket" to correct location for Alpine linux
Stop rspamd writing its own pid file

diff -Nurp a/conf/options.inc b/conf/options.inc
--- a/conf/options.inc	2020-05-10 21:06:38.577103479 +0100
+++ b/conf/options.inc	2020-05-16 06:41:17.587431999 +0100
@@ -44,7 +44,7 @@ classify_headers = [
 	"X-MimeOLE",
 ];
 
-control_socket = "$DBDIR/rspamd.sock mode=0600";
+control_socket = "$RUNDIR/rspamd.sock mode=0600";
 history_rows = 200;
 explicit_modules = ["settings", "bayes_expiry"];
 
diff -Nurp a/conf/rspamd.conf b/conf/rspamd.conf
--- a/conf/rspamd.conf	2020-05-10 21:06:38.577103479 +0100
+++ b/conf/rspamd.conf	2020-05-16 06:47:44.117319223 +0100
@@ -18,7 +18,6 @@
 .include "$CONFDIR/common.conf"
 
 options {
-    pidfile = "$RUNDIR/rspamd.pid";
     .include "$CONFDIR/options.inc"
     .include(try=true; priority=1,duplicate=merge) "$LOCAL_CONFDIR/local.d/options.inc"
     .include(try=true; priority=10) "$LOCAL_CONFDIR/override.d/options.inc"
